File: VB/PersonalOrganizer/ViewModels/MainViewModel.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Mvvm
Imports DevExpress.Mvvm.DataAnnotations

Namespace PersonalOrganizer.ViewModels
    <POCOViewModel> _
    Public Class MainViewModel
        Protected Overridable ReadOnly Property CurrentWindowService() As ICurrentWindowService
            Get
                Return Nothing
            End Get
        End Property
        Public Sub [Exit]()
            CurrentWindowService.Close()
        End Sub
    End Class
End Namespace

File: CS/PersonalOrganizer/ViewModels/MainViewModel.cs
using DevExpress.Mvvm;
using DevExpress.Mvvm.DataAnnotations;

namespace PersonalOrganizer.ViewModels {
    [POCOViewModel]
    public class MainViewModel {
        protected virtual ICurrentWindowService CurrentWindowService { get { return null; } }
        public void Exit() {
            CurrentWindowService.Close();
        }
    }
}